doc/documentation_data_format.md

```diff
@@ -366,9 +366,42 @@ Additional columns may be added.
     - parameterScaleLaplace: location; scale
 
 
-## Parameter estimation problems combining multiple models
+### Additional optional columns
 
-[**Issue #49**](https://github.com/ICB-DCM/PEtab/issues/49)
+Extra columns:
+
+- `hierarchicalOptimization` (optional)
+
+  hierarchicalOptimization: 1 if parameter is optimized using hierarchical
+  optimization approach. 0 otherwise.
+
+- `initializationPriorType` (optional)
+
+  Prior types used for sampling of initial point for optimization. Uses the
+  entries from `priorType` as default, but will overwrite those, if
+  something else is specified here. For more detailed documentation, see
+  `priorType`.
+
+- `initializationPriorParameters` (optional)
+
+  Prior parameters used for sampling of initial point for optimization. Uses
+  the entries from `priorParameters` as default, but will overwrite those, if
+  something else is specified here. For more detailed documentation, see
+  `priorParameters`.
+
+- `objectivePriorType` (optional)
+
+  Prior types used for the objective function during optimization. Uses the
```

Comment:
Note: This prior is presumably not only used in optimization, but also in sampling.